packages/mojo-drupal-scaffold/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "bluehorndigital/mojo-drupal-scaffold",
+    "descripton": "Scaffolding to add some Mojo to your Drupal projects.",
+    "type": "composer-plugin",
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "composer-plugin-api": "^1.1 || ^2.0",
+        "drupal/core-composer-scaffold": "^9.0",
+        "vlucas/phpdotenv": "^5.3"
+    },
+    "conflict": {
+        "amazeeio/drupal_integrations": "*"
+    },
+    "autoload": {
+        "psr-4": {
+            "BluehornDigital\\Mojo\\": "src"
+        }
+    },
+    "extra": {
+        "class": "BluehornDigital\\Mojo\\DrupalScaffoldPlugin",
+        "drupal-scaffold": {
+            "file-mapping": {
+                "[project-root]/.env.example": "assets/.env.example",
+                "[project-root]/load.environment.php": "assets/load.environment.php",
+                "[project-root]/phpunit.xml.dist": {
+                    "mode": "replace",
+                    "path": "assets/phpunit.xml.dist",
+                    "overwrite": false
+                },
+                "[web-root]/sites/default/settings.redis.php": "assets/settings.redis.php",
+                "[web-root]/sites/default/settings.php": {
+                    "mode": "replace",
+                    "path": "assets/mojo.settings.php",
+                    "overwrite": false
+                },
```

Comment:
Always overwrite, but update root `drupal-core-project-message` to suggest setting this to `false` if customizing.